modules/navier_stokes/test/tests/finite_volume/two_phase/mixture_model/channel-drift-flux-w-interface-area.i

```diff
@@ -0,0 +1,368 @@
+mu = 10.0
```

Comment:
We need a test for this input file. Something very simple (decrease mesh resolution) to make sure it remains runnable no matter who changes what.

Reply:
Added test for input file.